merfi/backends/rpm_sign.py

```diff
@@ -57,18 +60,19 @@ def sign(self):
             raise RuntimeError('specify a --key for signing')
         logger.info('Starting path collection, looking for files to sign')
         repos = RepoCollector(self.path)
```

Comment:
The purpose of the `merfi rpm-sign` backend is to allow interaction with Red Hat's internal `/usr/bin/rpm-sign` tool. The `/usr/bin/rpm-sign` utility was originally developed to sign RPMs (high-level description at http://www.awe.com/mark/blog/200701300906.html), but it was expanded so that it can sign arbitrary content with the `--detached` or `--clearsign` options.

So the tool name "rpm-sign" is sort of a poor name at this point :( My intention is that `merfi rpm-sign` could be used to sign Debian repositories or RPM files+repositories. (And then eventually make `merfi gpg` do that, too).